lit/docs/resources/managing.lit

```diff
@@ -28,3 +28,42 @@
   \italic{bosh-io-release} resource might use \code{version:11.2} in place of
   \code{ref:abcdef}.
 }
+
+\section{
+  \title{\code{fly pin-resource}}{fly-pin-resource}
+
+  To pin a resource to a specific version of that resource, run:
+
+  \codeblock{bash}{{{
+  $ fly -t example pin-resource --resource my-pipeline/my-resource \
+      --version ref:bceaf
+  }}}
+
+  Note that the version needs to be provided as key-value pairs. For the git
+  resource the \code{ref:} prefix is used. For example, the \italic{registry} 
+  resource might use \code{digest} as a prefix like \code{digest:sha256:94be7d7b}.
+
+  A comment can be provided using the \code{--comment} flag,
+  which is then also visible in the UI :
+
+  \codeblock{bash}{{{
+  $ fly -t example pin-resource --resource my-pipeline/my-resource \
+      --version ref:abcdef \
+      --comment "Some reason"
+  }}}
+
+  This can for example be used to pull in a fixed version of an external dependency
+  which might break your build in a new release. After the problem has been resolved
+  the pin can then be removed again.
```

Comment:
i got confused because i thought this meant the pin had already been removed once
```suggestion
  the pin can be removed.
```